packages/docs/getting-started/running-attestation-service.md

```diff
@@ -56,39 +57,19 @@ If you prefer using Twilio, this is list of the variables to use:
 
 ## Accounts Configuration
 
-First we need to create an account for getting the attestation key needed to sign the attestations. Run:
+First, the validator account needs to authorize the attestation signer. With `celocli` connected to a node that has the attestation signer unlocked, let's create a proof-of-posession:
 
 ```bash
-celocli account:new
+celocli account:proof-of-possession --signer $ATTESTATION_SIGNER_ADDRESS --account $CELO_VALIDATOR_ADDRESS
 ```
 
-We copy the account details and assign the Private Key to the `ATTESTATION_PRIVATE_KEY` environment variable:
+That will give you a signature that you can then use to authorize the key (with `celocli` connected to the node that has the validator account unlocked):
```

Comment:
The generated proof-of-possession can be used to authorize the attestations signer. This authorization needs to come from the Validator account. On the machine where you have access to the Validator account, run the following command.